packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -106,6 +106,37 @@ new codepipeline_actions.CodeBuildAction({
 });
 ```
 
+#### BitBucket
+
+CodePipeline can use a BitBucket Git repository as a source:
+
+**Note**: you have to manually connect CodePipeline through the AWS Console with your BitBucket account.
+This is a one-time operation for a given AWS account in a given region.
+The simplest way to do that is to either start creating a new CodePipeline,
+or edit na existing one, while being logged in to BitBucket.
+Choose BitBucket as the source,
+and grant CodePipeline permissions to your BitBucket account.
+Copy & paste the Connection ARN that you get in the console,
+or use the [`codestar-connections list-connections` AWS CLI operation](https://docs.aws.amazon.com/cli/latest/reference/codestar-connections/list-connections.html)
+to find it.
+After that, you can safely abort creating or editing the pipeline -
+the connection has already been created.
```

Comment:
This is a pretty lousy user experience... Is there a plan to improve this?

Reply:
I plan to reach out to the team owning this to talk about a way to make it automated.